<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameWidget</class>
 <widget class="QWidget" name="NewGameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Game</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="KComboBox" name="FirstMoving">
       <item>
        <property name="text">
         <string>First Player</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Second Player</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KComboBox" name="GameMode">
       <property name="autoCompletion">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Default Mode</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra Turn</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="HeightSpinBox">
       <property name="minimum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>First Moving</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Game Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="WidthSpinBox">
       <property name="minimum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
